source/regression1.md

```diff
@@ -807,15 +807,22 @@ To assess how well our model might do at predicting on unseen data, we will
 assess its RMSPE on the test data. To do this, we first need to retrain the 
 KNN regression model on the entire training data set using $K =$ {glue:}`best_k_sacr`
 neighbors. Fortunately we do not have to do this ourselves manually; `scikit-learn`
-does it for us. We just need to obtain the `best_estimator_` attribute of original
+does it for us automatically. If we want to see which parameters were identified
```

Comment:
this edit breaks the flow -- we previously had this here because we said
- you now need to retrain with best param
- scikit does it for you
- here is the object to use below that has the best params

But I agree that just calling `.predict` on the fit object is far more natural.

I'm going to move the `best_params_` up to right after we plot the CV results -- there it makes sense since we can show people how to grab the best result without visually inspecting the plot

Reply:
I think we should also add something similar to classification 2 -- i can handle that

Reply:
(with corresponding change to R version)